### DISCUSSION TITLE
Link 2 traces

### DISCUSSION THREAD

**Discussion by saiprasath21** [langfuse/langfuse#3557] [Support] [answered]
I have 2 traces: one from running my RAG chain and another from scoring the results using ragas. I want to store the trace_id of the rag chain in the scoring trace and vice versa. So that I can move between these 2 traces when analyzing them. 

NOTE: One possible solution is to store the trace_ids in the metadata, but then they won't be clickable in the UI. I want a way to click on the trace_id and move between this pair.

**Comment by marcklingen** [comment, marked as answer]
I'd recommend to include the trace url of the other trace. See docs on this here: https://langfuse.com/docs/tracing-features/url
